Source/Plugins/Core/com.equella.core/src/com/tle/web/lti/usermanagement/LisLtiWrapperExtension.java

```diff
@@ -32,11 +33,29 @@ public String getUserId(HttpServletRequest request) {
     return null;
   }
 
+  @Override
+  public String getUserId(HttpServletRequest request, String param) {
```

Comment:
same idea as canvas and d2l - for this LIS LTI wrapper, can we ignore the `param`?